go/private/actions/archive.bzl

```diff
@@ -58,9 +58,13 @@ def emit_archive(go, source = None, _recompile_suffix = "", recompile_internal_d
         pre_ext += _recompile_suffix
     out_lib = go.declare_file(go, name = source.library.name, ext = pre_ext + ".a")
 
-    # store __.PKGDEF and nogo facts in .x
+    # store export information for compiling dependent packages separately
     out_export = go.declare_file(go, name = source.library.name, ext = pre_ext + ".x")
```

Comment:
Nit: it would be nice to have these 2 renamed as well. 
Im thinking `s/out_lib/out_archive/` and `s/out_export/out_linkobj/`.

Could be done in a separate PR though to avoid noise.